### PR TITLE
Yep 66 search button

### PR DESCRIPTION
**Pull request by filveith**
# THIS IS NOT THE FINAL VERSION
I just wanted to push this, to be able to test it monday,

### CODE REVIEW

File: front/src/components/Search/SearchBar.tsx [hunk 1]
```diff
@@ -1,8 +1,100 @@
...
   return (
-    <div className="w-full flex justify-center">
-      <h1>Search Bar</h1>
-    </div>
+    <Container className="w-90 flex justify-center flex-col mt-4">
+      <form onSubmit={form.onSubmit((values: any) => onSubmit(values))}>
+        <Accordion
+          mx="auto"
+          variant="filled"
+          defaultValue="customization"
```
Comment (iittooss) [inline front/src/components/Search/SearchBar.tsx:68]: you can remove this if you don't want the search accordion to open

Comment (iittooss) [reply front/src/components/Search/SearchBar.tsx:68]: to open by default*

File: front/src/components/Search/SearchBar.tsx [hunk 2]
```diff
@@ -1,8 +1,100 @@
...
+          classNames={classes}
+          className={classes.root}
+        >
+          <Accordion.Item value="customization" className="pb-4">
+            <Accordion.Control>
```
Comment (iittooss) [inline front/src/components/Search/SearchBar.tsx:73]: i think you can move the search out of the control. When you leave the input, and go back in the accordion will close/open each time. https://mantine.dev/core/accordion/#compose-controls

Clearly it's not a big problem or a priority

Comment (iittooss) [reply front/src/components/Search/SearchBar.tsx:73]: same thing for the genre accordion

File: front/src/components/Search/SearchBar.tsx (hunk 1 above)
Comment (iittooss) [reply front/src/components/Search/SearchBar.tsx:68]: if you do this, you can't have access to the search button, in french will call this "bien baisé" :upside_down_face:

File: front/src/components/GigzAccordion.tsx [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+  header,
+  children,
+  className,
+}) => {
+  const { classes } = useStyles();
+  return (
+    <Accordion
+      mx="auto"
+      variant="filled"
+      defaultValue="customization"
```
Comment (iittooss) [inline front/src/components/GigzAccordion.tsx:58]: here view the search bar comment

File: front/src/components/Search/Filters/GenreSelector.tsx [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+  useEffect(() => {
+    form.values.genres = selectedGenre;
+  }, [selectedGenre]);
+
+  const handleAddGenre = (id: number) => {
+    if (!selectedGenre.includes(id)) {
+      setSelectedGenre((old: number[]) => [...old, id]);
+    } else {
+      setSelectedGenre((old: number[]) =>
+        old.filter((idGenre) => idGenre != id)
```
Comment (iittooss) [inline front/src/components/Search/Filters/GenreSelector.tsx:27]: not "!==" but "!=" (I have done the same thing in my genre step :disappointed: )

File: front/src/components/Search/Result/ResultList.tsx [hunk 5]
```diff
@@ -34,27 +31,26 @@ const SerachResultList: React.FC = () => {
...
+          <Result
             key={result.id}
             result={result}
             onClick={selectResult}
             loading={loading}
           />
         ))}
-        <div className="flex justify-center pb-20 pt-5">
+        <div className="flex justify-center pt-5 mb-40">
           <Text c="dimmed">{t('search.list.endOfProfiles')}</Text>
```
Comment (iittooss) [inline front/src/components/Search/Result/ResultList.tsx:44]: to change if there is no result ?

File: front/src/components/Search/SearchBar.tsx (hunk 2 above)
Comment (filveith) [reply front/src/components/Search/SearchBar.tsx:73]: The input is not opening on click now but the genres will open on click but not close when clicking on the input

**Review by iittooss — APPROVED** [review]

**Review by ASparton — APPROVED** [review]

**Pull request merged**